fulp_modules/features/ert/code/ert_outfit.dm

```diff
@@ -13,9 +13,8 @@
 /datum/outfit/centcom/ert/commander/medical
 	name = "ERT Commander - Medical"
 
-	suit = /obj/item/clothing/suit/space/hardsuit/ert/commandermed
 	suit_store = /obj/item/gun/energy/e_gun
-	back = /obj/item/storage/backpack/ert/medical
+	back = /obj/item/mod/control/pre_equipped/fulp/ert/commander/medical
```

Comment:
is this a bag? and it'll have all items from backpack_contents?

Reply:
Yeah thats how it works